InventoryIndexer/Plugin/InventorySales/ScheduleAfterPlaceReservationsForSalesEvent.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryIndexer\Plugin\InventorySales;
+
+use Magento\InventorySales\Model\PlaceReservationsForSalesEvent;
+use Magento\Framework\MessageQueue\PublisherInterface;
+
+/**
+ * Invalidate Inventory Indexer after Source was enabled or disabled.
+ */
+class ScheduleAfterPlaceReservationsForSalesEvent
+{
+
+    public const TOPIC_RESERVATION_PLACED = "inventory.reservation.place";
+
+    /**
+     * @var PublisherInterface
+     */
+    private $publisher;
+
+    /**
+     * @param PublisherInterface $publisher
+     */
+    public function __construct(
+        PublisherInterface $publisher
+    ) {
+        $this->publisher = $publisher;
+    }
+
+    /**
+     * Schedule stock status update after resevation placed
+     *
+     * @param PlaceReservationsForSalesEvent $subject
+     * @param void $result
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function afterExecute(
+        PlaceReservationsForSalesEvent $subject,
+        $result
+    ) {
+        $this->publisher->publish(self::TOPIC_RESERVATION_PLACED, []);
```

Comment:
we should send the subject here.  Stock or SalesChannel and SKU, as the handler has to have these data to check the salability status.

From that perspective, the append reservation is more suitable. But not we are not looking for easy ways

Reply:
See https://github.com/magento/inventory/pull/3026